boards/FSM/Inc/App/App_AcceleratorPedal.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
```

Comment:
Later on, accelerator pedal will have things like `GetPedalPosition` and what not. Keeping the interface with just enough methods to work with `Signal`.